spec/controllers/replies_controller_spec.rb

```diff
@@ -275,6 +275,7 @@
       other_user = create(:user)
       login_as(user)
       reply_post = create(:post, user: other_user, tagging_authors: [user, other_user], authors_locked: true)
+      reply_post.mark_read(user)
```

Comment:
What's the goal here? It makes me nervous that we need this side effect.

Reply:
The post/reply split means the post-reply isn't marked read. For some reason. Possibly it _should_ be? But I think the current marking-read occurs b/c of the page being loaded after?
Anyway if this shouldn't be fixed elsewhere this can safely be merged before that, which was the critera I was pulling things out from that branch by. It can have it's own tho I suppose?